enterprise/server/tasksize/tasksize.go

```diff
@@ -31,6 +31,8 @@ var (
 	psiCorrectionFactor = flag.Float64("remote_execution.task_size_psi_correction", 1.0, "What percentage of full-stall time should be subtracted from the execution duration.")
 	cpuQuotaLimit       = flag.Duration("remote_execution.cpu_quota_limit", 30*100*time.Millisecond /*30 cores*/, "Maximum CPU time allowed for each quota period.")
 	cpuQuotaPeriod      = flag.Duration("remote_execution.cpu_quota_period", 100*time.Millisecond, "How often the CPU quota is refreshed.")
+	memoryLimitBytes    = flag.Int64("remote_execution.memory_limit_bytes", 0, "Task cgroup memory limit in bytes.")
```

Comment:
I don't have any concrete plans for setting this flag in production yet, but I had to set this to something low-ish like 8GB locally to avoid all the apps on my machine getting OOM-killed while testing :sweat_smile: 

I do think we might want to set this flag to something reasonable eventually, though.